### PR TITLE
change nunique

### PR DESCRIPTION
**Pull request by Rubtsowa**

### CODE REVIEW

File: hpat/datatypes/hpat_pandas_series_functions.py [hunk 1]
```diff
@@ -1517,18 +1517,18 @@ def hpat_pandas_series_nunique(self, dropna=True):
...
-                [3, np.nan, 'b', 'b', 5.1, np.nan, 'ddd', 'dd']
             """
 
             str_set = set(self._data)
-            return len(str_set)
- 
+            if dropna == False:
+                return len(str_set)
+            else:
+                return len(str_set) + 1
```
Comment (shssf) [inline hpat/datatypes/hpat_pandas_series_functions.py]: This should not work with Numba pipeline (in HPAT these tests are disabled).
Input data is String, non None exists and parameter dropna=True
To do such thing I did `nan_exists` variable below

Comment (Rubtsowa) [reply hpat/datatypes/hpat_pandas_series_functions.py]: @shssf Shouldn't it work at all or is it implemented incorrectly?

Comment (shssf) [reply hpat/datatypes/hpat_pandas_series_functions.py]: @Rubtsowa I would expect failed tests

**Comment by Rubtsowa** [comment]
@shssf I made changes. Now the error, it seems, does not occur. Check, please.

**Pull request merged**